app/controllers/children_controller.rb

```diff
@@ -1,21 +1,27 @@
 class ChildrenController < ApplicationController
   before_action :authenticate_user!
 
-  helper_method :child, :children, :family
-
-  def index; end
+  def index
+    @children = current_partner.children.order(active: :desc, last_name: :asc)
```

Comment:
Any chance this will be used somewhere else? Is it worth extracting it to the children model layer?

Reply:
Good point. I think ultimately we will want to add scopes and filtering here similar to diaperbase, but I think its ok to punt on for now.